tests/test_de_serialisation.py

```diff
@@ -40,6 +41,28 @@ def test_anon_str_serialize():
     assert result_json == expected_json_updated
 
 
+def test_datetime_serialize():
+    # No param
+    example_constraints = {"birthdays": DateTimeTransformer()}
+    result_json = serialise_constraints(example_constraints)
+    expected_json = """{"module": "smartnoise-synth", "version": "1.0.4", "constraints": {"birthdays": {"type": "_ssynth_datetime_transformer:DateTimeTransformer", "params": {"epoch": "1970-01-01T00:00:00"}}}}"""  # noqa
+    expected_json_updated = expected_json.replace(
+        "1.0.4", pkg_resources.get_distribution(SSYNTH).version
```

Comment:
nice! i guess we could use the same logic for the measurement pipeline in our test with opendp. Instead of updating each time we use a new version.

Reply:
ah yes ^^ let's be lazy